server/src/routes/item.ts

```diff
@@ -0,0 +1,57 @@
+import { Request, Response, Router } from "express";
+import validateData from "../middleware/dataValidation";
+import {
+  createItem,
+  getAllItems,
+  getOneItem,
+  updateOneItem,
+} from "../controller/item";
+import { validateUUID } from "../middleware/validateUUID";
+import { itemCreateValidator } from "../validators";
+
+const router: Router = Router();
+
+router.post(
```

Comment:
According with swagger, when the user creates a new item, he/she should send an array of tags that are item related, so we will need to validate that we are receiving an array of strings ["tag1", "tag2", "tag3", ..., "tagN"] and the app should know it is a new tag and create the tag and if it exists, it should just have to connect both items